airflow/jobs.py

```diff
@@ -34,7 +34,7 @@
 from time import sleep
```

Comment:
We probably want to remove the DEADLOCK_CHECK_DEP_CONTEXT logic that we added along with this.